### PR TITLE
test: (e2e) add e2e tests for Notification component 

### PR DESCRIPTION
**Pull request by nikvalor**
#### Please provide a link to the associated issue.
closes#https://github.com/SAP/fundamental-ngx/issues/5212
#### Please provide a brief summary of this pull request.
Added notification core e2e tests

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by droshev — CHANGES_REQUESTED** [review]
We are about to refactor the components so let's hold on those tests

File: e2e/wdio/core/tests/notification.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+        it('verify notification creation and all elements are displayed', () => {
+            scrollIntoView(notificationButton);
+            click(notificationButton);
+            const blockLength = getElementArrayLength(notificationBlock);
+            for (let i = 0; i < blockLength; i++) {
+                scrollIntoView(notificationBlock, i);
+                expect(isElementDisplayed(notificationBlock, i)).toBe(true, `block with index ${i} not displayed`);
+                expect(isElementDisplayed(notificationAvatar, i)).toBe(true, `avatar with index ${i} not displayed`);
+                expect(isElementDisplayed(notificationIndicator, i)).toBe(true, `indicator with index ${i} not displayed`);
+                expect(isElementClickable(buttonMoreInfo, i)).toBe(true, `button more with index ${i} not clickable`);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/notification.e2e-spec.ts:51]: 'button more with index..' is hard to understand. better: 'more info button with index... '

File: e2e/wdio/core/tests/notification.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+    describe('Check Template as Content example', function() {
+
+        it('verify notification creation and all elements are displayed', () => {
+            scrollIntoView(notificationButton, 1);
+            click(notificationButton, 1);
+            scrollIntoView(notificationBlock);
+            expect(isElementDisplayed(notificationBlock)).toBe(true, `block with index not displayed`);
+            expect(isElementDisplayed(notificationAvatar)).toBe(true, `avatar with index not displayed`);
+            expect(isElementClickable(buttonMoreInfo)).toBe(true, `button more with index not clickable`);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/notification.e2e-spec.ts:109]: same comment as above

File: e2e/wdio/core/tests/notification.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+    describe('Check Component as Content example', function() {
+
+        it('verify notification creation and all elements are displayed', () => {
+            scrollIntoView(notificationButton, 2);
+            click(notificationButton, 2);
+            scrollIntoView(notificationBlock);
+            expect(isElementDisplayed(notificationBlock)).toBe(true, `block with index not displayed`);
+            expect(isElementDisplayed(indicatorSuccess)).toBe(true, `indicator with index not displayed`);
+            expect(isElementDisplayed(notificationAvatar)).toBe(true, `avatar with index not displayed`);
+            expect(isElementClickable(buttonMoreInfo)).toBe(true, `button more with index not clickable`);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/notification.e2e-spec.ts:146]: same as above

File: e2e/wdio/core/tests/notification.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+    describe('Check Notification Groups example', function() {
+
+        it('verify notification creation and all elements are displayed', () => {
+            scrollIntoView(notificationButton, 3);
+            click(notificationButton, 3);
+            scrollIntoView(notificationBlock);
+            expect(isElementDisplayed(notificationBlock)).toBe(true, `block with index not displayed`);
+            expect(isElementDisplayed(indicatorWarning)).toBe(true, `indicator with index not displayed`);
+            expect(isElementDisplayed(notificationAvatar)).toBe(true, `avatar with index not displayed`);
+            expect(isElementClickable(buttonMoreInfo)).toBe(true, `button more with index not clickable`);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/notification.e2e-spec.ts:183]: same as above

File: e2e/wdio/core/tests/notification.e2e-spec.ts [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+    describe('Check Notifications From Object example', function() {
+
+        it('verify notification creation and all elements are displayed', () => {
+            scrollIntoView(notificationButton, 4);
+            click(notificationButton, 4);
+            scrollIntoView(notificationBlock);
+            expect(isElementDisplayed(notificationBlock)).toBe(true, `block with index not displayed`);
+            expect(isElementDisplayed(indicatorSuccess)).toBe(true, `indicator with index not displayed`);
+            expect(isElementDisplayed(notificationAvatar)).toBe(true, `avatar with index not displayed`);
+            expect(isElementClickable(buttonMoreInfo)).toBe(true, `button more with index not clickable`);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/notification.e2e-spec.ts:220]: same as above

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

**Review by AntonOlkhovskyi — APPROVED** [review]

**Review by Seamoo13 — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
@nikvalor are these e2e tests for the updated Notifications?

**Pull request closed without merging**